controllers/awscluster_controller.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/pkg/errors"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
+	capi "sigs.k8s.io/cluster-api/api/v1alpha3"
```

Comment:
could we use a more recent api version?

Reply:
I think we will need to do this for the whole controller, but that is out of the scope of this PR, 
bumping  the version but breaks the import and it wont build

I will drop a note to the board